prometheus-exporters/install-exporter-amd64.sh

```diff
@@ -197,6 +213,22 @@ WantedBy=multi-user.target
 EOF
   fi
 
+  if [ "$EXPORTER" == "nginx" ]; then
+    cat << EOF > $EXPORTER_SERVICE_FILE
+[Unit]
+Description=Prometheus Nginx Exporter
+
+[Service]
+User=root
+ExecStart=/usr/local/bin/nginx-prometheus-exporter -nginx.scrape-uri=http://localhost/basic_status
```

Comment:
For the default config, shall we please use `:8080` as the port to expose `stub_status`?  In fact, the default URL is `http://127.0.0.1:8080/stub_status` - so if we recommend the customer to expose this on 8080 and under `/stub_status` then, we don't have to set this arg.

And if the customer exposes in any other url, then they can set `SCRAPE_URI` env var.